pdns/pdns_recursor.cc

```diff
@@ -637,6 +637,20 @@ static void protobufLogResponse(const std::shared_ptr<RemoteLogger>& logger, con
 }
 #endif
 
+void handleRPZCustom(const DNSRecord& spoofed, const QType& qtype, SyncRes& sr, int& res, vector<DNSRecord>& ret)
+{
+  if (spoofed.d_type == QType::CNAME) {
+    sr.d_wantsRPZ = false;
```

Comment:
I considered this, but realized when we hit RPZ we always send out the answer (and never use the SyncRes again). But to prevent future surprises, this is a good idea.